webapp/channels/src/root.html

```diff
@@ -19,68 +19,13 @@
     <link rel="icon" type="image/png" href="/static/images/favicon/favicon-default-64x64.png" sizes="64x64">
     <link rel="icon" type="image/png" href="/static/images/favicon/favicon-default-96x96.png" sizes="96x96">
 
-    <!-- CSS Should always go first -->
-    <link rel='stylesheet' class='code_theme'>
-    <style>
-        .error-screen {
-            font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
-            padding-top: 50px;
-            max-width: 750px;
-            font-size: 14px;
-            color: #333333;
-            margin: auto;
-            display: none;
-            line-height: 1.5;
-        }
-
-        .error-screen h2 {
-            font-size: 30px;
-            font-weight: normal;
-            line-height: 1.2;
-        }
-
-        .error-screen ul {
-            padding-left: 15px;
-            line-height: 1.7;
-            margin-top: 0;
-            margin-bottom: 10px;
-        }
-
-        .error-screen hr {
-            color: #ddd;
-            margin-top: 20px;
-            margin-bottom: 20px;
-            border: 0;
-            border-top: 1px solid #eee;
-        }
-
-        .error-screen-visible {
-            display: block;
-        }
-    </style>
+    <!-- We need this CSS first to show the initial loading screen -->
+    <link id="initialLoadingScreenCSS" rel='stylesheet' href='/static/css/initial_loading_screen.css'>
 </head>
 <body class='font--open_sans enable-animations'>
-    <div id='root'>
-        <div class='error-screen'>
```

Comment:
Not used anymore https://github.com/mattermost/mattermost-webapp/commit/d09ea8d47df82cbdfc50b233d093c6f32874e477